group4/81966_Georgi_Gerginov/Set/worker_log/main.cpp

```diff
@@ -0,0 +1,14 @@
+#include "log/log.h"
+#include "set/set.h"
+
+int main() {
+	size_t num_of_logs;
```

Comment:
The idea is a little bit different - enter the work logs into list to preserve all the data, then make a set to extract the unique values.